### DISCUSSION TITLE
npm run build Error

### DISCUSSION THREAD

**Discussion by Gvalien** [twbs/bootstrap#39073] [Q&A] [answered]
got error in Laravel project 
i have installed: 
npm install sass --save-dev   
npm install bootstrap@5.3.1 --save-dev

npm run build                         

> build
> vite build

vite v4.4.9 building for production...
More info: https://sass-lang.com/d/abs-percent

   ╷
57 │   $dividend: abs($dividend);
   │              ^^^^^^^^^^^^^^
   ╵
    node_modules\bootstrap\scss\vendor\_rfs.scss 57:14         divide()
    node_modules\bootstrap\scss\mixins\_grid.scss 59:12        row-cols()
    node_modules\bootstrap\scss\mixins\_grid.scss 85:13        @content
    node_modules\bootstrap\scss\mixins\_breakpoints.scss 68:5  media-breakpoint-up()
    node_modules\bootstrap\scss\mixins\_grid.scss 72:5         make-grid-columns()
    node_modules\bootstrap\scss\_grid.scss 38:3                @import
    node_modules\bootstrap\scss\bootstrap.scss 20:9            @import
    resources\scss\app.scss 1:9                                root stylesheet

**Comment by JamesVanWaza** [comment, marked as answer]
@Gvalien 
They are aware of the issue see - https://github.com/twbs/bootstrap/issues/39028
